packaging/rpm/microshift.spec

```diff
@@ -126,6 +126,13 @@ Requires: greenboot
 %description greenboot
 The microshift-greenboot package provides the Greenboot scripts used for verifying that MicroShift is up and running.
 
+%package microshift-olm-manifests
```

Comment:
I think we probably want this to be simply `microshift-olm`? Is there an obvious benefit to specifying "manifests"?

Reply:
Done!